spec/results/proto-aws-iam-profile.yml

```diff
@@ -353,57 +400,55 @@ networks:
   version: 271.0.0
 - name: bpm
-  sha1: c956394fce7e74f741e4ae8c256b480904ad5942
-  url: https://bosh.io/d/github.com/cloudfoundry/bpm-release?v=1.1.8
+  sha1: d9a22dff24a659a8f6585a600fdd7e27bf211f3d
+  url: https://s3.amazonaws.com/bosh-compiled-release-tarballs/bpm-1.1.8-ubuntu-xenial-621.71-20200422-220353-760960221-20200422220355.tgz
   version: 1.1.8
 - name: uaa
-  sha1: ad4967615f2321648372497402ca6359ab8ba0df
-  url: https://bosh.io/d/github.com/cloudfoundry/uaa-release?v=74.17.0
+  sha1: 186514cb390f6be7eb39a6ce861c99b0dc87f80b
+  url: https://s3.amazonaws.com/bosh-compiled-release-tarballs/uaa-74.17.0-ubuntu-xenial-621.71-20200422-215929-649183041-20200422215932.tgz
   version: 74.17.0
 - name: credhub
-  sha1: c45af16ed393bb3cf061b8749e3ee4cae74ce995
-  url: https://bosh.io/d/github.com/pivotal-cf/credhub-release?v=2.6.0
+  sha1: 479420f5ec04945b5dc76db955588bcd7603ff01
+  url: https://s3.amazonaws.com/bosh-compiled-release-tarballs/credhub-2.6.0-ubuntu-xenial-621.71-20200422-215930-454454003-20200422215931.tgz
   version: 2.6.0
 - name: bosh-aws-cpi
   sha1: 1a4826469e715f5595de38a15df7b7f511fbfe85
   url: https://bosh.io/d/github.com/cloudfoundry/bosh-aws-cpi-release?v=82
-  version: "82"
+  version: 82
 - name: os-conf
-  sha1: 7579a96515b265c6d828924bf4f5fae115798199
-  url: https://bosh.io/d/github.com/cloudfoundry/os-conf-release?v=21.0.0
-  version: 21.0.0
+  sha1: 6946056ad69ae378cb89c9ef76daf66370a7dc6a
+  url: https://bosh.io/d/github.com/cloudfoundry/os-conf-release?v=22.0.0
+  version: 22.0.0
 resource_pools:
 - cloud_properties:
     availability_zone: test-regiona
     ephemeral_disk:
-      size: 25600
+      size: 25000
       type: gp2
-    iam_instance_profile: test-profile-name
-    instance_type: m4.large
+    instance_type: m5.xlarge
   env:
     bosh:
+      blobstores:
+      - options:
+          blobstore_path: /var/vcap/micro_bosh/data/cache
+        provider: local
       mbus:
         cert:
           ca: <!{meta.vault}/ssl/ca:certificate!>
           certificate: <!{meta.vault}/ssl/mbus:certificate!>
           private_key: <!{meta.vault}/ssl/mbus:key!>
-      password: <!{meta.vault}/vcap:password-crypt-sha512!>
-  name: large
-  network: bosh
+      ntp:
+      - 169.254.169.123
+      password: '*'
+  name: vms
+  network: default
   stemcell:
     sha1: f2b54011eb4891f86638294c9f2d8670b04e371b
-    url: https://bosh.io/d/stemcells/bosh-aws-xen-hvm-ubuntu-xenial-go_agent?v=621.71
-stemcells:
-- alias: default
-  os: ubuntu-xenial
-  version: 621.latest
-update:
-  canaries: 1
-  canary_watch_time: 30000-600000
-  max_in_flight: 1
-  serial: true
-  update_watch_time: 30000-600000
+    url: https://bosh-aws-light-stemcells.s3-accelerate.amazonaws.com/621.71/light-bosh-stemcell-621.71-aws-xen-hvm-ubuntu-xenial-go_agent.tgz
+- cloud_properties:
+    iam_instance_profile: test-profile-name
```

Comment:
shouldn't this be merged with the other entry in resource_pools?

Reply:
@dennisjbell it seems they set it with name `bosh_vm_type` which is large in this case.
the default name is `vms`.
if i read this correctly it should be under `vms`